src/core/geometry/qgsgeometry.h

```diff
@@ -880,8 +880,19 @@ class CORE_EXPORT QgsGeometry
      * \param topological TRUE if topological editing is enabled
      * \param[out] topologyTestPoints points that need to be tested for topological completeness in the dataset
      * \returns OperationResult a result code: success or reason of failure
+     * \deprecated in QGIS 3.12 - will be removed in QGIS 4.0
```

Comment:
```suggestion
     * \deprecated in QGIS 3.12 - will be removed in QGIS 4.0. Use the variant which accepts QgsPoint objects instead of QgsPointXY instead.
```
(same for other deprecated methods -- gives users a hint how to handle this deprecation)

Reply:
Done, thanks.